packages/edit-site/src/components/page-pages/index.js

```diff
@@ -339,6 +339,22 @@ export default function PagePages() {
 					view={ view }
 					onChangeView={ onChangeView }
 					onSelectionChange={ onSelectionChange }
+					labels={ {
```

Comment:
Can we find a way to avoid this prop, find generic labels or something?

Reply:
The generic labels would need to be something like "Select a new item" or "Deselect item", we could not even reference the title because the dataviews component itself is not aware of what field is a "title". We already have this as a fallback if the labels are not passed.
Not describing the item you are selecting on the label seems like a11y regression compared to the current page PHP list in core. I think we will need this prop or at least a getItemTitle prop (similar to getItemId) that provides a title of the row so we could at least do something like "Select item: %s", where %s is the page or template title. The labels property is more descriptive and differentiates templates and pages but I would also be ok with a  getItemTitle prop and something more generic, let me know what option you would prefer.

Reply:
I think what bothers me the most is that these are functions, maybe we should keep the "sprintf" call within the component and just make these regular strings.

Reply:
in other words:

```
labels={ { selectItem: __( 'Select page: %s' ), deselectItem: __( 'Deselect page: %s' ) } }
```

Reply:
I applied the suggestion.